src/frontend/filter/index.js

```diff
@@ -77,6 +77,21 @@ const Filter = () => {
 					</Button>
 				</>
 			) }
+			<div className="wporg-meeting-calendar__filter-feed">
+				<Button
+					icon="calendar-alt"
+					isSecondary
+					style={ {
+						marginLeft: 'auto',
+					} }
+					onClick={ ( e ) => {
+						window.open( '/meetings.ics' );
```

Comment:
This will work fine on make.wordpress.org but the URL should account for the WordPress root possibly being in a subdirectory.